src/components/ZclEndpointCard.vue

```diff
@@ -394,8 +427,9 @@ export default {
   },
   created() {
     if (this.$serverGet != null) {
-      this.selectedAttributes= []
-      this.selectedReporting =[]
+      this.selectedservers = []
```

Comment:
typo selectedservers?

Reply:
No, sometime it's break a part of code. after adding this line we don't have any issue

Reply:
Fixed